tests/lametro/test_events.py

```diff
@@ -30,4 +30,28 @@ def test_status_assignment(event_scraper,
         mocker.patch('lametro.LametroEventScraper._merge_events', return_value=[(api_event, web_event)])
 
         for event in event_scraper.scrape():
-            assert event.status == scraper_assigned_status
\ No newline at end of file
+            assert event.status == scraper_assigned_status
+
+def test_sequence_duplicate_error(event_scraper,
+                                  api_event, 
+                                  web_event,
+                                  event_agenda_item,
+                                  mocker):
+    with requests_mock.Mocker() as m:
+        matcher = re.compile('webapi.legistar.com')
+        m.get(matcher, json={}, status_code=200)
+
+        matcher = re.compile('metro.legistar.com')
+        m.get(matcher, json={}, status_code=200)
+
+        api_event['event_details'] = {'note': 'web', 
+                                      'url': 'https://metro.legistar.com/MeetingDetail.aspx?ID=642118&GUID=F19B2133-928C-4390-9566-C293C61DC89A&Options=info&Search='}
+
+        mocker.patch('lametro.LametroEventScraper._merge_events', return_value=[(api_event, web_event)])
+        mocker.patch('lametro.LametroEventScraper.agenda', return_value=[event_agenda_item, event_agenda_item])
```

Comment:
nice. :-) actual question: should we also test that no duplicates are scraped without error? (or is this a given?)

Reply:
Oh, yes: we should. I got lazy.

Reply:
you: lazy once. me: proactively lazy, potentially many times. ha.